cpp/src/arrow/pretty_print_test.cc

```diff
@@ -200,6 +200,53 @@ TEST_F(TestPrettyPrint, PrimitiveTypeNoNewlines) {
   CheckPrimitive<Int32Type, int32_t>(options, is_valid, values, expected, false);
 }
 
+TEST_F(TestPrettyPrint, PrimitiveTypeCustomArrayElementDelimiter) {
+  PrettyPrintOptions options{};
+  // Display array contents on one line.
+  options.skip_new_lines = true;
+  // Display maximum of 3 elements at the beginning and at the end of the array
+  options.window = 3;
```

Comment:
Hmm, that's a good point.

I included these because I wanted to emulate the behavior of how we would like the MATLAB display to work (i.e. horizontal display with no new lines and a small window), but I am happy to remove these to keep the test more focused.

Reply:
I understand your thought.
I think that we can remove them because we will be able to detect whether mixing these options is broken or not in MATLAB tests.
Anyway, we can keep these options here if you want. :-)

Reply:
Fixed.

Reply:
**Update**: I realized I only updated one of the tests before to not modify the value of `skip_new_lines` or `window`. I just made a change to update the other test. Sorry for missing this before.